test/dicer-write.test.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const { test } = require('tap')
+const { Dicer } = require('../lib/main')
+
+test('dicer _write method', t => {
+  t.plan(1)
+
+  t.test('calls the callback cb() when headerFirst is set and all the data have been written', t => {
+    const dicer = new Dicer({ headerFirst: true })
+
+    dicer._write(Buffer.from('Content-Type: text/plain'), null, () => {
+      dicer._write(Buffer.from('Content-Type: text/plain'), null, () => {
+        t.pass('write method called')
+      })
+    })
+
+    t.end()
```

Comment:
Basically not testing anything, because you call t.end().  use t.plan for this subtest also.

Reply:
@Uzlopak oh, good to know! First time using this testing framework!

Reply:
remove the t.end() please

Reply:
@Uzlopak is there anything else for me to do in this PR?